components/support/nimbus-fml/src/backends/kotlin/templates/FeatureManifestTemplate.kt

```diff
@@ -95,6 +95,15 @@ object {{ nimbus_object }} : FeatureManifestInterface<{{ nimbus_object }}.Featur
         {%- endfor %}
     }
 
+    /**
+     * Get a list of feature ids where the feature allows co-enrollment.
+     */
+    override fun getCoenrollingFeatureIds(): List<String> = listOf(
```

Comment:
When this is generated into `app.fml_release.kt`, `simple_nimbus_validation_release.kt`, etc, it's generating this:

```
override fun getCoenrollingFeatureIds(): List<String> = listOf()
```

Is that right? Are these files just showing the default (which should just be `listOf<string>()`), or should it be showing the actual logic (aka calling the feature manifest's `get_coenrolling_feature_ids()`) 🤔

Reply:
Nope if that's the default then I think that's working correctly! Unlike the FFI places in the Nimbus SDK where we do call back into the Rust, the templates won't. The FML templates may generate code that would call into _SDK_ Rust code, but they'll never generate code that would call into FML Rust code.

Reply:
Another way you can think about this is it's very similar to rendering a React/Angular component's template. The template itself has a bunch of references into component code, but you never actually see the JS that's inside the template because it all gets rendered into HTML.

Reply:
That totally makes sense! Ty! 🙏